app/src/main/java/co/nimblehq/compose/crypto/ui/preview/HomeScreenPreviewParameterProvider.kt

```diff
@@ -1,23 +1,22 @@
 package co.nimblehq.compose.crypto.ui.preview
 
 import androidx.compose.ui.tooling.preview.PreviewParameterProvider
-import co.nimblehq.compose.crypto.lib.IsLoading
-import co.nimblehq.compose.crypto.ui.base.LoadingState
-import co.nimblehq.compose.crypto.ui.uimodel.CoinItemUiModel
+import co.nimblehq.compose.crypto.core.lib.IsLoading
+import co.nimblehq.compose.crypto.core.uimodel.CoinItemUiModel
 
 class HomeScreenPreviewParameterProvider : PreviewParameterProvider<HomeScreenParams> {
     override val values = sequenceOf(
         HomeScreenParams(
             listOf(coinItemPreview),
             listOf(coinItemPreview),
             false,
-            LoadingState.Idle
+            co.nimblehq.compose.crypto.core.LoadingState.Idle
```

Comment:
We should not use the full path here. We should import `import co.nimblehq.compose.crypto.core.LoadingState` instead.
```suggestion
            LoadingState.Idle
```